src/test/e2e/helpers/test-helper.ts

```diff
@@ -321,8 +320,6 @@ async function createApp(
     });
     const services = createServices(stores, config, db);
     const unleashSession = sessionDb(config, undefined);
-    const emitter = new EventEmitter();
-    emitter.setMaxListeners(0);
```

Comment:
Was this code doing anything? Config contains an event emitter and this variable is not used at all

Reply:
doesn't look like it was no. Good catch

Reply:
I've seen some errors in logs which I believe are prevented by this line. I'll re-add it later with a comment